packages/big-design-icons/svgs/material/info.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 15c-.55 0-1-.45-1-1v-4c0-.55.45-1 1-1s1 .45 1 1v4c0 .55-.45 1-1 1zm1-8h-2V7h2v2z"/></svg>
```

Comment:
I see that this is the same icon as an `ErrorIcon`. I think it's better to leave one component instead of duplication.

I propose to rename ErrorIcon to InfoIcon. What do you think?

Reply:
The info icon is different than the error icon. One is an `!` while the other is a `i`.
<img width="39" alt="Screen Shot 2020-02-18 at 10 52 08 AM" src="https://user-images.githubusercontent.com/10539418/74758360-d9929880-523c-11ea-9cc8-c4ada1b12d39.png">
vs. 
<img width="32" alt="Screen Shot 2020-02-18 at 10 52 12 AM" src="https://user-images.githubusercontent.com/10539418/74758362-da2b2f00-523c-11ea-877d-1e54a6cac212.png">

Reply:
got it, sorry, my bad!